spec/features/user_search_spec.rb

```diff
@@ -7,6 +7,7 @@
 
   describe 'without filter' do
     let!(:users) { FactoryBot.create_list(:user, 3) }
+    let(:second_user) { FactoryBot.create :user }
```

Comment:
Is this user still needed? It is the only change in this file, so I assume that it can be omitted?
```suggestion
```